src/stories/LossSelection.stories.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { ComponentMeta } from '@storybook/react';
+import HistorySelection, { HistorySelectionProps } from './HsitorySelect';
+
+export default {
+    component: HistorySelection,
+    title: 'History Selection/HistorySelection',
+} as ComponentMeta<typeof HistorySelection>;
+
+function Template(args: HistorySelectionProps) {

```

Comment:
create interface with color prop and it's data type and use it for Template component